hpolib/container/recipes/ml/Singularity.XGBoostBenchmark

```diff
@@ -11,7 +11,7 @@ VERSION v0.0.1
     && cd /home \
     && git clone https://github.com/automl/HPOlib3.git \
     && cd HPOlib3 \
-    && git checkout master \
+    && git checkout development \
```

Comment:
Why? For the benchmarks we are done with we should point to master.

Reply:
ah! sorry! my bad.